department-of-veterans-affairs/caseflow
# Appeals 5832 Disable Convert to Virtual button until required fields are populated

Resolves #[APPEALS-5832](https://vajira.max.gov/browse/APPEALS-5832)
**Description**
As a VSO user I'd like the Convert to Virtual command to  be disabled until all required fields are populated on the Hearings form so that consistent behavior is observed across both forms.

**Acceptance Criteria**

- [ ] The Convert to Virtual button is disabled until the following form fields are populated (Convert to Virtual form - Hearings Application)
    - <Veteran/Appellant> email
    - <Veteran/Appellant> timezone
    - POA/Representative timezone

**Testing Plan**
1. Create an intake and schedule an appeal at least 11 days into the future.
2. Switch to a VSO user and click on the "Convert to virtual" link in the hearings section on the case details page.
3. The button should be disabled until the Appellant Email, Appellant Timezone, and Representative Timezone are filled.
